### PR TITLE
Split binary/symbol and then upload ortsrv nightly build to blob storage

### PR DESCRIPTION
**Pull request by NonStatic2014**

### CODE REVIEW

File: tools/ci_build/build.py [hunk 1]
```diff
@@ -826,6 +839,9 @@ def main():
               if args.use_mkldnn:
                 run_onnx_tests(build_dir, configs, onnx_test_data_dir, 'mkldnn', True, 1)
 
+    if args.build_server:
```
Comment (pranavsharma) [inline tools/ci_build/build.py]: Can this step be done as the last step of the pipeline definition? Why does build.py need to know about this?

Comment (NonStatic2014) [reply tools/ci_build/build.py]: As we discussed offline, the `objcopy` is only available in the build environment. We have to install some package if we want to do it as downloading test data.

Comment (NonStatic2014) [reply tools/ci_build/build.py]: Also, I would expect the `RelWithDebInfo` is defined to split binary and symbol sometimes. So we may consider do this for all builds in this config.

**Pull request merged**